src/Altitude.sol

```diff
@@ -44,7 +57,7 @@ contract Altitude is Ownable2Step {
 
     function forceInclude(address receiver, uint[2] calldata leafCoins, uint highestCoinToSend, bytes memory signature) external {
         require(highestCoinToSend < maxCoin, ERROR_FORCE_NO_COIN);
-        uint hash = PoseidonT5.hash([leafCoins[0], leafCoins[1], highestCoinToSend, uint(uint160(receiver))]);
+        uint hash = PoseidonT5.hash([signature, highestCoinToSend, uint(uint160(receiver))]);
```

Comment:
Sorry this is a half baked idea! I have reverted it because it's not related to this PR

Is signature included in the hash the user wants to sign? The `hash` variable here just includes the receiver and various coin numbers. I think it's particularly important to explicitly include the signature in the `forceInclusions` if we're allowing invalid signatures in the circuit, so we can force the operator to use the correct signature.

Also, the leaf shouldn't change if a user is trying to force a transaction. If a leaf changes, that means the user signed a transaction to spend those coins, and they wouldn't do that if they wanted to force a transaction.